packages/hdwallet-ledger/src/ledger.ts

```diff
@@ -322,25 +322,24 @@ export class LedgerHDWallet implements core.HDWallet, core.BTCWallet, core.ETHWa
   }
 
   /**
-   * Validates current app open
-   * Throws WrongApp error if symbol does not match app
-   * @param symbol - i.e. 'BCH'
-   * @returns {Promise<void>}
+   * Validate if a specific app is open
+   * Throws WrongApp error if app associated with coin is not open
+   * @param coin  Name of coin for app name lookup
    */
-  public async validateCurrentApp (symbol: string): Promise<void> {
-    if (!symbol) return
+  public async validateCurrentApp (coin: core.Coin): Promise<void> {
+    if (!coin) return
 
-    const expectedApp = appsUtil[symbol.toUpperCase()]
-    if (!expectedApp) {
-      throw new Error(`Cannot find app for symbol: ${symbol}`)
+    const { appName = undefined } = networksUtil[core.slip44ByCoin(coin)] || {}
```

Comment:
the `... || {}` part of this assignment makes me think - we would never get `undefined` as default.

Reply:
Updated with lodash get